database/index.js

```diff
@@ -14,7 +14,7 @@ mongoose.connect('mongodb://localhost/bonuses', {
 
 // Get bonus associated with bundle
 const getBonuses = (bundleId, callback) => {
-  Soundtrack.find({ bundle_id: bundleId }, (err, data) => {
+  Soundtrack.find({ bundle_id: bundleId }, { _id: 0, 'bonus_info._id': 0, 'bonus_info.tracklist._id': 0 }, (err, data) => {
```

Comment:
What's the purpose of this added object here, and the extended objects on lines 29 and 41?

Reply:
Mongo adds it's automated mega `_id` to every object as it is added and I hate looking at it and don't plan on referencing it. The added object is a query filter that takes out all of the MongoDB `_id`s before returning the query results.